### PR TITLE
HPCC-29440 Enable ESP service method execution profiling by default in debug builds

### PR DESCRIPTION
**Pull request by kenrowland**
Enabled ESP service method execution profiling in debug builds by default

Signed-Off-By: Kenneth Rowland kenneth.rowland@lexisnexisrisk.com

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [x] All new and existing tests passed.
  - [x] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: esp/services/ws_fileio/CMakeLists.txt [hunk 1]
```diff
@@ -61,7 +61,7 @@ include_directories (
          ${HPCC_SOURCE_DIR}/dali/sasha
     )
 
-ADD_DEFINITIONS( -D_USRDLL -DWsFileIO_API_LOCAL )
+ADD_DEFINITIONS( -D_USRDLL -DWsFileIO_API_LOCAL -DESP_CONTROL_WsFileIO)
```
Comment (afishbeckln) [inline esp/services/ws_fileio/CMakeLists.txt:64]: @kenrowland Why are some of these "ESP_CONTROL_" when the others are "ESP_SERVICE_"?

**Comment by afishbeckln** [comment]
@kenrowland the commit message and the PR title and description could use reformatting.

**Review by afishbeck — COMMENTED** [review]
@kenrowland a couple comments.

File: esp/services/ws_fileio/CMakeLists.txt (hunk 1 above)
Comment (kenrowland) [reply esp/services/ws_fileio/CMakeLists.txt:64]: It should be ESP_SERVICE. Good catch. Probably from having to type it in so many times.

**Comment by kenrowland** [comment]
@afishbeck Updated the commit message (not sure why it split it the way it did). When it is rebased I'll make sure the message is correct as well.

**Review by afishbeck — APPROVED** [review]

**Comment by kenrowland** [comment]
@ghalliday please merge

**Review by ghalliday — APPROVED** [review]

**Pull request merged**